frontend/components/GameSpoilersList.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+// eslint-disable-next-line no-unused-vars
+import {AppStore} from '../storage/common';
+// eslint-disable-next-line no-unused-vars
+import { GameSpoilers } from '../storage/models/GameSpoilers';
+interface GameSpoilersListProps{
+	gameSpoilers: Map<number, GameSpoilers>;
+}
+const GameSpoilersList: React.FC<GameSpoilersListProps> = (props: GameSpoilersListProps) =>{
+//I couldn't do destructuring, because i get following error: 'Property 'gameSpoilers' does not exist on type 'Map<number, GameSpoilers>''
+	const gameSpoilers = props.gameSpoilers;
```

Comment:
Did you have it as `const {gameSpoilers} = props` or `const {gameSpoilers} = props.gameSpoilers`?

Reply:
Oh well, it was props.GameSpoilers. Now I see my error :)